openIMIS/openIMIS/git_scripts.py

```diff
@@ -0,0 +1,112 @@
+import git
+import json
+import os
+
+
+from os import path
+from django.conf import settings
+from openIMIS.openimisapps import openimis_apps
+
+
+def create_release_branches_backend(version):
+    """
+       function to create release branches 'release/<version>'
+       for all backend modules presented in openimis.json
+    """
+    output_messages = []
+    # modules = openimis_apps()
+    # TODO temporary value for modules for test
+    modules = ['calculation_rule-fs_income_percentage', 'report', 'policyholder', 'contribution_plan', 'calculation']
+    directory = path.abspath(path.join(settings.BASE_DIR, "../.."))
```

Comment:
Is this for reaching parent directory? If so, can you use settings.BASE_DIR.parent instead of hardcoded "../.."?

Reply:
Yes. Exactly. I think your proposition with 'parent' is better.